shopinvader_api_cart/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    applied_transaction_uuids = fields.Char(
+        readonly=True
+    )  # TODO: this field is only required for API's. Where to put it?
```

Comment:
I don't know what to do with this field and if it should stay here.
It is a char field that only records, on a cart, all transaction uuids that were performed. These transaction uuid are given in the `sync` services. This field is not used anywhere else, it is only used for some tracking purposes I guess.

Reply:
IMO, you can leave it here

Reply:
It's used to know which transactions have been processed by odoo

Reply:
Done along with a migration script